api/v1alpha1/openapi.yaml

```diff
@@ -390,6 +390,145 @@ paths:
           description: NotFound
         "500":
           description: Internal Server Error
+  /api/v1/sources/{id}/share-token:
+    post:
+      tags:
+        - share
+      description: Create a new share token for a source or return existing one
+      operationId: createShareToken
+      parameters:
+        - name: id
+          in: path
+          description: ID of the source
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ShareToken'
```

Comment:
Just wonder why we have to return here full object with all those paramters, if user at the end of the day use just token, to get the source, and `id` to delete the token? Also is it a responsibility of user to remember both to remove/get it?

Reply:
I think we need to discuss further what we want to achieve here. 
My impression was that once the share token URL is used, the user with whom the source was shared would have access to the shared assessment using the normal UI (e.g. accessing the UI in the normal way, the user would see the new shared assessments along with their own).
Am I missing something here?

Reply:
If I am not mistaken as current design, user won't see it withing other sources. When user enter shared URL like `https://console.redhat.com/openshift/migration-assessment/source/{token}` he will be redirected to overview page and he will be able to see just the overview, when he goes back he will not see the source anymore, unless he enter the shared URL again. So if he looses the URL he losses the access. Is that right, Nir?

Reply:
I have changed the schema to return only the source ID and the shared token